interaction_lib.py

```
@@ -0,0 +1,307 @@
+        p_value = 1 - chi2.cdf(lr_stat, df)
+        
+        return {
+            'lr_stat': lr_stat,
+            'df': df,
+            'p_value': p_value,
+            'improvement': p_value < 0.05,
+            'aic_base': result_base.aic,
+            'aic_full': result_full.aic,
+            'bic_base': result_base.bic,
+            'bic_full': result_full.bic
+        }
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)